extensions/management.py

```diff
@@ -13,6 +13,37 @@ async def topic(self, ctx, description):
         await ctx.channel.edit(topic=description)
         await ctx.send("Channel topic updated")
 
+    # Listener pour la création des salons vocaux de réunion (et leur supression)
+    @commands.Cog.listener()
+    async def on_voice_state_update(self, member, before, after):
+        # Création
+        try:
+            if (
+                after.channel.name
+                == self.bot.config["voicechannelcreation"]["triggerchannel_name"]

```

Comment:
- can you use snake case standard voicechannelcreation => voice_channel_creation etc
- don't forget to edit default config.example in the repository with new settings